pkg/rpc/client/nep11.go

```diff
@@ -46,9 +46,6 @@ func (c *Client) NEP11TokenInfo(tokenHash util.Uint160) (*wallet.Token, error) {
 // given account and sends it to the network returning just a hash of it.
 func (c *Client) TransferNEP11(acc *wallet.Account, to util.Uint160,
 	tokenHash util.Uint160, tokenID string, data interface{}, gas int64, cosigners []SignerAccount) (util.Uint256, error) {
-	if !c.cache.initDone {
-		return util.Uint256{}, errNetworkNotInitialized
```

Comment:
But the method won't work correctly if the client is not initialized.

Reply:
It's true, but `TransferNEP11` itself doesn't use client's cache, it is done by internal helper. And this helper has the `initDone` check and will return an error if the client is not initialized. After that `TransferNEP11` should just handle this error.

The idea of this commit is to remove these extra `initDone` checks and to keep them only in places where direct access to cache is occurred. We can always return an error from these methods, which IMO is preferable.